resources/guis.yml

```diff
@@ -224,3 +224,13 @@ guis:
   price: Free
   license: GNU GPL
   order: 25
+- name: GitBlade
+  url: https://gitblade.com
+  image_tag: guis/gitblade@2x.png
+  platforms:
+    - Linux
```

Comment:
thanks for fixing this :)

I don't want to sound picky, but I just saw on the website that currently only Linux is supported (Mac and Windows coming soon™). In order to keep this list useful, I would prefer to add only Linux support now, and the remaining later, when supported. Would you be ok with that?

Reply:
Sure, totally makes sense, I'll fix it.

Reply:
Ok, this is fixed.